libr/include/r_vector.h

```diff
@@ -158,6 +160,16 @@ static inline bool r_pvector_empty(RPVector *vec) {
 	return r_pvector_len (vec) == 0;
 }
 
+// returns a pointer to the offset inside the array where the element of the index lies.
+static inline void **r_pvector_index_ptr(RPVector *vec, size_t index) {
+	return ((void **)vec->v.a) + index;
```

Comment:
whhat if index exceeds thee vector size, what about returning *left?

Reply:
What is "left"? It would be the caller's responsibility to check the size here.